webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx

```diff
@@ -160,14 +160,14 @@ export default class LinkingLandingPage extends PureComponent<Props, State> {
     renderGoNativeAppMessage = () => {
         return (
             <a
-                href={Utils.isMobile() ? '#' : this.state.nativeLocation}
```

Comment:
The changes in this file are a slight change of behaviour, but  I think this is correct since we care about the platform with these links, not desktop/mobile view in the web app